versioned_docs/version-2.9/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

```diff
@@ -232,3 +232,13 @@ To upgrade legacy multi-cluster apps:
 ## Limitations
 
 Dashboard apps or Rancher feature charts can't be installed using the Rancher CLI.
+
+## Known Bugs 
+
+As of now, the **Apps > Installed Apps** page displays the version in `Upgradable` column by combining the versions of the chart from all the Helm repositories installed in Rancher.
```

Comment:
Has this always worked this way or is it a regression? If it's not a regression, maybe we can move it to the `Limitations` section and frame it as something that's hasn't been implemented yet as opposed to a bug.